### PR TITLE
PLAT-64158: Fix Popup to prevent duplicated 5-way navigation

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`moonstone/Popup` components handle focus movement redundantly. 
In `moonstone/Popup` case with two or more buttons, focus is moved twice at one 5way-key input.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
In `moonstone/Popup` keydown event handler, it called `Spotlight.move(dir)` without stopPropagation() . So `Spotlight` handle the key event redundantly.

`moonstone/Popup` is fixed to prevent additional key event handling after focus changed using stopPropagation() and preventDefault(). 
And added more button to verify the spotlight operation in Popup story of qa-sampler .

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

Since the focus has been moved, the event could be regarded as consumed. Thus I added preventDefault() and stopPropagation(). But I don't have confidence it need preventDefault() because when I checked several additional cases(include scroller case), there was no difference in the behavior.

### Links
[//]: # (Related issues, references)
PLAT-64158

### CODE REVIEW

**Comment by 0x64** [comment]
LGTM

File: packages/moonstone/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/Popup` to prevent moving twice at one 5-way key input when the `Popup` has `spotlightRestrict='self-first'` option and there is two or more button
```
Comment (ybsung) [inline packages/moonstone/CHANGELOG.md]: there is two or more button
=> there are two or more buttons

**Comment by ybsung** [comment]
@jeonghee27 
Please fix merge conflict.

**Comment by ybsung** [comment]
LGTM

**Pull request merged**